operators_sample_website/website/app/GBlib/Repositories/IGbCasRestServices.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\GBlib\Repositories;
+
+interface IGbCasRestServices
+{
+
+    const CAS_URL = 'https://192.168.51.117:7743'; // your Crypto application server url
```

Comment:
Replace by something different e.g. 127.0.0.1 or nothing if possible? Just to force user to fill in some value